hazelcast/src/main/java/com/hazelcast/internal/partition/operation/PartitionBackupReplicaAntiEntropyOperation.java

```diff
@@ -63,9 +66,35 @@ public void run() {
         int partitionId = getPartitionId();
         int replicaIndex = getReplicaIndex();
 
+        InternalPartitionImpl partition = partitionService.getPartitionStateManager().getPartitionImpl(partitionId);
+        int currentReplicaIndex = partition.getReplicaIndex(PartitionReplica.from(getNodeEngine().getLocalMember()));
+
+        ILogger logger = getLogger();
+        if (replicaIndex != currentReplicaIndex) {
+            logger.fine("Anti-entropy operation for partitionId=" + getPartitionId() + ", replicaIndex=" + getReplicaIndex()
```

Comment:
probably guard with a check if `fine` level is enabled? (similarly below)

Reply:
I didn't add an `if` guard here because this is a very edge case, this is not printed often. During testing I've just seen these logs a few times only.